== TEST (2019-06-12 13:31:53) 
-- CODE:
from numpy import*
x = array(eval(input("vetor: ")))

t = 0
k = 0

for i in x:
	if (i % 2) == 0:
		t = t
	else:
		t = t + 1
t = zeros(t, dtype=int)
for j in x:
	if (i % 2) != 0:
		t[k] = x[i]
		k = k + 1
	elif:
		k = k
print(t)
-- ERROR:
File "XXXX", line 17
    elif:
        ^
SyntaxError: invalid syntax
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-06-12 13:32:00) 
-- CODE:
from numpy import*
x = array(eval(input("vetor: ")))

t = 0
k = 0

for i in x:
	if (i % 2) == 0:
		t = t
	else:
		t = t + 1
t = zeros(t, dtype=int)
for j in x:
	if (i % 2) != 0:
		t[k] = x[i]
		k = k + 1
	else:
		k = k
print(t)
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    from numpy import*
ModuleNotFoundError: No module named 'numpy'
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-06-12 13:33:42) 
-- CODE:
from numpy import*
x = array(eval(input("vetor: ")))

t = 0
k = 0

for i in x:
	if (i % 2) == 0:
		t = t
	else:
		t = t + 1
t = zeros(t, dtype=int)
for j in x:
	if (i % 2) != 0:
		t[k] = x[j]
		k = k + 1
	else:
		k = k
print(t)
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    from numpy import*
ModuleNotFoundError: No module named 'numpy'
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-06-12 13:34:38) 
-- CODE:
from numpy import*
x = array(eval(input("vetor: ")))

t = 0
k = 0

for i in x:
	if (i % 2) == 0:
		t = t
	else:
		t = t + 1
t = zeros(t, dtype=int)
for j in x:
	if (j % 2) != 0:
		t[k] = x[j]
		k = k + 1
	else:
		k = k
print(t)
-- EXECUTION TIME:
1.04175
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1798/2572/main.py", line 15, in <module>
    t[k] = x[j]
IndexError: index 75247561 is out of bounds for axis 0 with size 5

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-06-12 13:35:13) 
-- CODE:
from numpy import*
x = array(eval(input("vetor: ")))

t = 0
k = 0

for i in x:
	if (i % 2) == 0:
		t = t
	else:
		t = t + 1
t = zeros(t, dtype=int)
for j in x:
	if (j % 2) != 0:
		t[k] = x[j]
		k = k + 1
print(t)
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    from numpy import*
ModuleNotFoundError: No module named 'numpy'
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-06-12 13:35:17) 
-- CODE:
from numpy import*
x = array(eval(input("vetor: ")))

t = 0
k = 0

for i in x:
	if (i % 2) == 0:
		t = t
	else:
		t = t + 1
t = zeros(t, dtype=int)
for j in x:
	if (j % 2) != 0:
		t[k] = x[j]
		k = k + 1
print(t)
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    from numpy import*
ModuleNotFoundError: No module named 'numpy'
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-06-12 13:35:17) 
-- CODE:
from numpy import*
x = array(eval(input("vetor: ")))

t = 0
k = 0

for i in x:
	if (i % 2) == 0:
		t = t
	else:
		t = t + 1
t = zeros(t, dtype=int)
for j in x:
	if (j % 2) != 0:
		t[k] = x[j]
		k = k + 1
print(t)
-- EXECUTION TIME:
0.912989
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1798/2572/main.py", line 15, in <module>
    t[k] = x[j]
IndexError: index 75247561 is out of bounds for axis 0 with size 5

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-06-12 13:35:53) 
-- CODE:
from numpy import*
x = array(eval(input("vetor: ")))

t = 0
k = 0

for i in x:
	if (i % 2) == 0:
		t = t
	else:
		t = t + 1
y = zeros(t, dtype=int)
for j in x:
	if (j % 2) != 0:
		y[k] = x[j]
		k = k + 1
print(t)
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    from numpy import*
ModuleNotFoundError: No module named 'numpy'
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-06-12 13:41:03) 
-- CODE:
from numpy import*
x = array(eval(input("vetor: ")))

t = 0
k = 0

for i in x:
	if (i % 2) == 0:
		t = t
	else:
		t = t + 1
y = zeros(t, dtype=int)
for j in x:
	if (j % 2) != 0:
		y[k] = x[j]
		k = k + 1
print(t)
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    from numpy import*
ModuleNotFoundError: No module named 'numpy'
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-06-12 13:41:58) 
-- CODE:
from numpy import*
x = array(eval(input("vetor: ")))

t = 0
k = 0

for i in x:
	if (i % 2) == 0:
		t = t
	else:
		t = t + 1
y = zeros(t, dtype=int)
for j in x:
	if (j % 2) != 0:
		y[k] = y[k] + x[j]
		k = k + 1
print(t)
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    from numpy import*
ModuleNotFoundError: No module named 'numpy'
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-06-12 13:45:01) 
-- CODE:
from numpy import*
x = array(eval(input("vetor: ")))

t = 0

for i in x:
	if (i % 2) == 0:
		t = t
	else:
		t = t + 1
print(t)
y = zeros(t, dtype=int)
k = 0
for j in x:
	if (j % 2) != 0:
		y[k] = y[k] + x[j]
		k = k + 1
print(y)
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    from numpy import*
ModuleNotFoundError: No module named 'numpy'
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-06-12 13:45:13) 
-- CODE:
from numpy import*
x = array(eval(input("vetor: ")))

t = 0

for i in x:
	if (i % 2) == 0:
		t = t
	else:
		t = t + 1
print(t)
y = zeros(t, dtype=int)
k = 0
for j in x:
	if (j % 2) != 0:
		y[k] = y[k] + x[j]
		k = k + 1
print(y)
-- EXECUTION TIME:
0.966491
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1798/2572/main.py", line 16, in <module>
    y[k] = y[k] + x[j]
IndexError: index 75247561 is out of bounds for axis 0 with size 5

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-06-12 13:45:22) 
-- CODE:
from numpy import*
x = array(eval(input("vetor: ")))

t = 0

for i in x:
	if (i % 2) == 0:
		t = t
	else:
		t = t + 1
y = zeros(t, dtype=int)
k = 0
for j in x:
	if (j % 2) != 0:
		y[k] = y[k] + x[j]
		k = k + 1
print(y)
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    from numpy import*
ModuleNotFoundError: No module named 'numpy'
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-06-12 13:45:25) 
-- CODE:
from numpy import*
x = array(eval(input("vetor: ")))

t = 0

for i in x:
	if (i % 2) == 0:
		t = t
	else:
		t = t + 1
y = zeros(t, dtype=int)
k = 0
for j in x:
	if (j % 2) != 0:
		y[k] = y[k] + x[j]
		k = k + 1
print(y)
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    from numpy import*
ModuleNotFoundError: No module named 'numpy'
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-06-12 13:45:27) 
-- CODE:
from numpy import*
x = array(eval(input("vetor: ")))

t = 0

for i in x:
	if (i % 2) == 0:
		t = t
	else:
		t = t + 1
y = zeros(t, dtype=int)
k = 0
for j in x:
	if (j % 2) != 0:
		y[k] = y[k] + x[j]
		k = k + 1
print(y)
-- EXECUTION TIME:
1.20169
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1798/2572/main.py", line 15, in <module>
    y[k] = y[k] + x[j]
IndexError: index 75247561 is out of bounds for axis 0 with size 5

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-06-12 13:47:46) 
-- CODE:
from numpy import*
x = array(eval(input("vetor: ")))

t = 0

for i in x:
	if (i % 2) == 0:
		t = t
	else:
		t = t + 1
y = zeros(t, dtype=int)
k = 0
for j in x:
	if (j % 2) != 0:
		y[k] = x[j]
		k = k + 1
print(y)
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    from numpy import*
ModuleNotFoundError: No module named 'numpy'
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-06-12 13:48:58) 
-- CODE:
from numpy import*
x = array(eval(input("vetor: ")))
z = x
t = 0

for i in x:
	if (i % 2) == 0:
		t = t
	else:
		t = t + 1
z = zeros(t, dtype=int)
k = 0
for j in z:
	if (j % 2) != 0:
		y[k] = z[j]
		k = k + 1
print(y)
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    from numpy import*
ModuleNotFoundError: No module named 'numpy'
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-06-12 13:49:16) 
-- CODE:
from numpy import*
x = array(eval(input("vetor: ")))
z = x
t = 0

for i in x:
	if (i % 2) == 0:
		t = t
	else:
		t = t + 1
z = zeros(t, dtype=int)
k = 0
for j in z:
	if (j % 2) != 0:
		y[k] = z[j]
		k = k + 1
print(z)
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    from numpy import*
ModuleNotFoundError: No module named 'numpy'
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-06-12 13:54:37) 
-- CODE:
from numpy import*
x = array(eval(input("vetor: ")))
z = x
t = 0

for i in x:
	if (i % 2) == 0:
		t = t
	else:
		t = t + 1
z = zeros(t, dtype=int)
k = 0
for j in z:
	if (j % 2) != 0:
		y[k] = j
		k = k + 1
print(z)
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    from numpy import*
ModuleNotFoundError: No module named 'numpy'
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-06-12 13:54:42) 
-- CODE:
from numpy import*
x = array(eval(input("vetor: ")))
z = x
t = 0

for i in x:
	if (i % 2) == 0:
		t = t
	else:
		t = t + 1
z = zeros(t, dtype=int)
k = 0
for j in z:
	if (j % 2) != 0:
		y[k] = j
		k = k + 1
print(z)
-- EXECUTION TIME:
1.02021
-- TEST CASE 1:
---- input:
[75247561,76546316,75713658,97135671,70356874]
---- correct output:
[75247561 97135671]
---- user output:
[0 0]
-- TEST CASE 2:
---- input:
[13017594,57417542,85024758,12475981,75942785,91875987,29584192,85794217]
---- correct output:
[12475981 75942785 91875987 85794217]
---- user output:
[0 0 0 0]
-- TEST CASE 3:
---- input:
[31874903,74902746,23742046]
---- correct output:
[31874903]
---- user output:
[0]
-- GRADE:
0%
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-06-12 13:54:43) 
-- CODE:
from numpy import*
x = array(eval(input("vetor: ")))
z = x
t = 0

for i in x:
	if (i % 2) == 0:
		t = t
	else:
		t = t + 1
z = zeros(t, dtype=int)
k = 0
for j in z:
	if (j % 2) != 0:
		y[k] = j
		k = k + 1
print(z)
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    from numpy import*
ModuleNotFoundError: No module named 'numpy'
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-06-12 13:55:03) 
-- CODE:
from numpy import*
x = array(eval(input("vetor: ")))
z = x
t = 0

for i in x:
	if (i % 2) == 0:
		t = t
	else:
		t = t + 1
z = zeros(t, dtype=int)
k = 0
for j in z:
	if (j % 2) != 0:
		z[k] = j
		k = k + 1
print(z)
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    from numpy import*
ModuleNotFoundError: No module named 'numpy'
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-06-12 13:55:25) 
-- CODE:
from numpy import*
x = array(eval(input("vetor: ")))
y = x
t = 0

for i in x:
	if (i % 2) == 0:
		t = t
	else:
		t = t + 1
z = zeros(t, dtype=int)
k = 0
for j in z:
	if (j % 2) != 0:
		y[k] = j
		k = k + 1
print(z)
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    from numpy import*
ModuleNotFoundError: No module named 'numpy'
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-06-12 13:56:54) 
-- CODE:
from numpy import*
x = array(eval(input("vetor: ")))
y = x
t = 0

for i in x:
	if (i % 2) == 0:
		t = t
	else:
		t = t + 1
z = zeros(t, dtype=int)
k = 0
for j in z:
	if (j % 2) != 0:
		z[k] = y[j]
		k = k + 1
print(z)
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    from numpy import*
ModuleNotFoundError: No module named 'numpy'
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-06-12 13:57:36) 
-- CODE:
from numpy import*
x = array(eval(input("vetor: ")))
y = x
t = 0

for i in x:
	if (i % 2) == 0:
		t = t
	else:
		t = t + 1
z = zeros(t, dtype=int)
k = 0
for j in y:
	if (j % 2) != 0:
		z[k] = y[j]
		k = k + 1
print(z)
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    from numpy import*
ModuleNotFoundError: No module named 'numpy'
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
